apps/docs/docs/state/actions/actions.mdx

```diff
@@ -256,99 +174,66 @@ This is also true for the programmatic usage in the component class or a service
 
 To ease this pain we can manage this login with `transforms`.
 
-You can write you own transforms or use the predefined functions:
+You can write your own transforms, leverage Browser APIs like `String` and `Boolean` or use the predefined functions.
 
-- preventDefault
-- stopPropagation
-- preventDefaultStopPropagation
-- eventValue
+### Use existing transform functions
+
+The existing transform functions are:
+
+- `preventDefault` - calls `preventDefault` on a passed event
+- `stopPropagation`
+- `preventDefaultStopPropagation`
+- `eventValue`
+
+Here we see how to use an action transforms. This concept is similar to Input transforms.
+The logic is placed in a single place and transforms the event before emission.
 
 ```typescript
-import { Component } from '@angular/core';
-import { RxState, RxActionFactory } from '@rx-angular/state';
-import { StateService } from './state.service';
-import { withLatestFrom, map, debounceTime } from 'rxjs/operators';
-
-interface UiActions {
-  submitBtn: void;
-  searchInput: string;
-}
+import { rxActions, eventValue } from '@rx-angular/state/actions';
+
+type ListUi = { searchInput: string };
 
 @Component({
-  selector: 'my-component',
-  template: `
-    <input (input)="ui.searchInput($event)" /> Search for:
-    {{ ui.searchInput$ | async }}<br />
-    <button (click)="ui.submitBtn()">Submit</button>
-    <br />
-    <ul>
-      <li *ngFor="let item of list$ | async">{{ item }}</li>
-    </ul>
-  `,
-  providers: [RxState, RxActionFactory],
+  // takes a DOM Event
+  template: `<input name="search" (change)="ui.searchInput($event)" />`,
 })
-export class MyComponent {
-  ui = this.factory.create<UiActions>({ searchInput: eventValue });
-  list$ = this.state.select('list');
-  submittedSearchQuery$ = this.ui.submitBtn$.pipe(
-    withLatestFrom(this.ui.searchInput$),
-    map(([_, search]) => search),
-    debounceTime(1500)
+class ListComponent {
+  protected ui = rxActions<ListUi>(({ transform }) =>
+    transform({ searchInput: eventValue })
```

Comment:
please add a comment on what `eventValue` actually does. For me it's not very clear